addons/binding/org.openhab.binding.tellstick/ESH-INF/thing/devices.xml

```diff
@@ -72,14 +72,14 @@
 				<description>The name of the device</description>
 				<required>false</required>
 			</parameter>
-			<parameter name="deviceId" type="integer">
+			<parameter name="deviceId" type="text">
 				<label>DeviceID</label>
 				<description>This is the unique id of a device.</description>
 				<required>true</required>
 			</parameter>
 
-			<parameter name="resend" type="integer">
-				<label>Resend</label>
+			<parameter name="repeat" type="integer">
```

Comment:
Good question, it actually looks incompatible. Also the integer->text change should have an impact on how textual configuration is done. In this context I wonder how all that is reflected in the [full example of the README.md](https://github.com/jarlebh/openhab2-addons/blob/80ebf71419481603bca7ced83f10b1838e7d9ec7/addons/binding/org.openhab.binding.tellstick/README.md#full-example), @jarlebh?